.github/workflows/publish-commit-bottles.yml

```diff
@@ -13,20 +13,30 @@ on:
         default: ""
       large_runner:
         description: "Whether to run the upload job on a large runner (default: false)"
+        type: boolean
         required: false
+        default: false
+      commit_bottles_to_pr_branch:
+        description: "Whether to push bottle commits to the pull request branch (default: false)"
+        type: boolean
+        required: false
+        default: false
```

Comment:
It turned out to be a bit fiddly to check the PR for labels, so let's use a workflow dispatch input instead. We can teach `brew pr-publish` to pass the right argument here if the PR has a given label if desired.